marxan.cpp

```diff
@@ -442,8 +447,8 @@ namespace marxan {
 
         delta = 1e-14;  // This would more elegantly be done as a constant
 
-        // initt global rng engine
-        rngEngine = mt19937(iseed);
+        // init rng engine

```

Comment:
I think originally the RANDSEED if set to -1 means we set a seed based on machine time (or something similar) to intentionally vary the results. It only applies when the value is > 0. I can't recall if this is still the case.

Reply:
rng_engine rngEngine(iseed); 
 below duplicate this functionality, where iseed  is read firm inputs.